pkg/client/unversioned/request.go

```diff
@@ -491,6 +491,7 @@ func (r *Request) Body(obj interface{}) *Request {
 		}
 		glog.V(8).Infof("Request Body: %s", string(data))
 		r.body = bytes.NewBuffer(data)
+		r.SetHeader("Content-Type", "application/json")
```

Comment:
This is a property of the encoding, but I guess it's fine right now since we only have one.